### PR TITLE
Update university-of-gothenburg-apa-swedish-legislations.csl

### PR DESCRIPTION
**Pull request by bjornekstrom**
DOI link fixed. Removed month and day from citations.

### CODE REVIEW

File: university-of-gothenburg-apa-swedish-legislations.csl [hunk 1]
```diff
@@ -129,7 +129,7 @@
       <if type="thesis report" match="any">
         <choose>
           <if variable="DOI" match="any">
-            <text variable="DOI" prefix="doi: "/>
+            <text variable="DOI" prefix="https://dx.doi.org/"/>
```
Comment (rmzelle) [inline university-of-gothenburg-apa-swedish-legislations.csl]: @bjornekstrom, do you know if https://doi.org/ would also be acceptable? "https://dx.doi.org/" as a prefix is actually already outdated, and the shorter https://doi.org/ is now the officially recommended format. See https://www.zotero.org/support/kb/doi_in_apa for details.

File: university-of-gothenburg-apa-swedish-legislations.csl [hunk 2]
```diff
@@ -151,7 +151,7 @@
       <else>
         <choose>
           <if variable="DOI">
-            <text variable="DOI" prefix="doi: "/>
+            <text variable="DOI" prefix="https://dx.doi.org/"/>
```
Comment (rmzelle) [inline university-of-gothenburg-apa-swedish-legislations.csl]: Idem.

File: university-of-gothenburg-apa-swedish-legislations.csl (hunk 1 above)
Comment (bjornekstrom) [reply university-of-gothenburg-apa-swedish-legislations.csl]: @rmzelle Yes, this works better. Thanks for your reply. I have made the necessary amendments now.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**